.env

```diff
@@ -15,3 +15,5 @@ SPORT_TYPES_TO_IMPORT='["Ride", "VirtualRide"]'
 ATHLETE_BIRTHDAY=replace-me
 ATHLETE_WEIGHTS='{}'
 FTP_VALUES='{}'
+PUID=65534
```

Comment:
These .env files are only used in local development. I guess users will need to add PUID and PGID to their docker-compose.yml file?

Reply:
They only need to use these if 1) they are on a linux host 2) they want to be able to edit generated files without needing sudo

Reply:
Removed these from .env's since they aren't really necessary. I had them here mostly for visibility